api/user_interaction_handler_test.go

```diff
@@ -0,0 +1,192 @@
+package api_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/topfreegames/pitaya-admin/api"
+	pt "github.com/topfreegames/pitaya-admin/testing"
+)
+
+func TestPushHandler(t *testing.T) {
+	t.Parallel()
+	if pt.IsConf == false {
+		pt.ConfApp()
+	}
+	a := pt.TestApp
+	var (
+		response    *httptest.ResponseRecorder
+		pushHandler = api.NewPushToUsersHandler(a)
+	)
+
+	tables := map[string]struct {
+		request *http.Request
+		asserts func(response *httptest.ResponseRecorder)
+	}{
+		"test_empty_body": {
+			request: func() *http.Request {
+				request, err := http.NewRequest("POST", "", nil)
```

Comment:
I think you are not POSTing to the correct route but I'm intrigued that this works.